synapse/lib/stormtypes.py

```diff
@@ -2128,6 +2127,9 @@ async def wput(self, sha256, url, headers=None, params=None, method='PUT', ssl=T
         params = self.strify(params)
         headers = self.strify(headers)
 
+        if proxy is not None:
+            self.runt.confirm(('storm', 'lib', 'axon', 'proxy'))
```

Comment:
I thought we were moving this to storm.lib.inet.http.proxy as a single permission for controlling proxy bypass?